index.html

```diff
@@ -13,140 +13,188 @@
     <header>
         <nav class="flex-nav"> 
             <a href="#" class="toggleNav">☰</a>
+            <ul class="menu">
+                <li>
+                    Hello
+                    <div class="line-hello"></div>
+                </li>
+                <li>Portfolio
+                    <div class="line-portfolio"></div>
+                </li>
+                <li>About
+                    <div class="line-about"></div>
+                </li>
+                <li>Contact
+                    <div class="line-contact"></div>
+                </li>
+            </ul>
         </nav>
     </header>
-    <main class="name">
-        <h1>Sebastián<br>Muñoz</h1>
+    <main>
+        <div class="main-text">
+            <div class="name">
+                <h1>Sebastián<br>Muñoz</h1>
+            </div>
+            <section class="intro">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```

Comment:
- [ ] Please remove the extra indentations between "website.          Take a ...".
Kindly notice that we're only adding blank lines and indentations for good reasons such as separating logical code or two large sections from each other.

> You can read more about this and other useful notes [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
fixed 👍